test/e2e/upgrade/upgrade.go

```diff
@@ -115,8 +115,8 @@ var _ = g.Describe("[Disruptive]", func() {
 			client := configv1client.NewForConfigOrDie(config)
 			dynamicClient := dynamic.NewForConfigOrDie(config)
 
-			upgCtx, err := getUpgradeContext(client, lifecycle.GetUpgradeTarget(), lifecycle.GetUpgradeImage())
-			framework.ExpectNoError(err, "determining what to upgrade to version=%s image=%s", lifecycle.GetUpgradeTarget(), lifecycle.GetUpgradeImage())
+			upgCtx, err := getUpgradeContext(client, gcp.GetUpgradeTarget(), gcp.GetUpgradeImage())
```

Comment:
I'm surprised this became cloud specific.  You sure?

Reply:
yes, https://github.com/kubernetes/kubernetes/commit/0b3d50b6dccdc4bbd0b3e411c648b092477d79ac#diff-3699a4098a3cce570a8d7d0a3a4b5574